### ISSUE TITLE
**Dataloader Use Cases**

### ISSUE THREAD

**Issue by robintibor** [braindecode/braindecode#2]
*Allow many people to use it*
- write down all different use cases (classification(single label/multi label)/regression/continuous labels etc.)
- describe how implementation could work in pytorch dataloader logic
